.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
```

Comment:
Just to satisfy my curiosity, do we need to keep an eye on the version numbers of these tools and update when necessary? Just curious

Reply:
I think so. @bdice has implemented some pretty slick stuff with pre-commit, so I'd love to hear his take here :)

Reply:
Actually--pre-commit autoupdates and will open a pull request: https://github.com/cmelab/grits/pull/7